main.cpp

```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bicycle
+//  action 1: the bicycle rides
+bicycle.ride();

```

Comment:
What is the bicycle riding?
Someone else rides the bicycle. 
how about `rolls`.?
"The bicycle rolls forward"
`bicycle.rollForward();`
- [ ] revise